src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let result = 40 * days;
+
```

Comment:
look checklist one more time please
https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md
1st - add please variables for main values: rent cost, discount, etc..